Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -151,23 +151,26 @@ namespace
 
             if (inputSource.JointsTrackedThisFrame)
             {
-                auto handJointCollection = Napi::Array::New(env, HAND_JOINT_NAMES.size());
+                auto handJointCollection = jsInputSource.Get("hand").As<Napi::Array>();
+                if (!jsInputSource.Has("hand"))
```

Comment:
if jsInputSource doesn't have a hand, won't the above line throw?

Reply:
i feel like a cast to napi array for undefined will cause a runtime error

Reply:
It doesn't seem to, but I'll change the logic to make it less scary